skimage/filters/rank/__init__.py

```diff
@@ -22,6 +22,7 @@
            'subtract_mean',
            'subtract_mean_percentile',
            'median',
+           'median_float',
```

Comment:
@spark1729 ``ndimage.median_filter`` has a different API, but you can still use the ``footprint`` and ``origin`` arguments to adapt the arguments passed to skimage's function ``median``.

Do you think you will have the time to modify your PR so that only one function ``median`` is exposed, inside which the dtype of the image is tested and a different function is called for uint8 (the existing code) and float types?